firmware/STM32/Core/Src/main.c

```diff
@@ -17,6 +17,7 @@
   */
 	#include "epc_conf.h"
 	#include "epc_st_err.h" //Import EPC_ST_ERR_COUNTER
+	extern uint8_t EPC_ST_ERR_COUNTER;
```

Comment:
Move these 3 lines to else block: below of include #include "main.h"

Reply:
moved except `#include "epc_conf.h"`